Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN chmod +x deploy/update-version.sh  \
     && deploy/update-version.sh version.yaml \
     && make swag \
     && CGO_ENABLED=0 go build -o main \
-    && make test
+    && make check
```

Comment:
Would this be possible as well? Removing `make swag` and swapping line 16 and 17.
If not, then why?

```Dockerfile
&& deploy/update-version.sh version.yaml \
&& make check \
&& CGO_ENABLED=0 go build -o main \
```

Reply:
Yea sure. I think I added it separately to have it explicit that we do both, but merging the two `make` calls will let make to skip the duplicate `swag` target runs.